test/resources/tekton/kn-pipeline-run.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: tekton.dev/v1alpha1
```

Comment:
Should the version be parametrized?

Reply:
If the tests fail with a newer version of Tekton then these files will need to be updated at that point and committed. The tests are there for testing regressions with future versions of Tekton / kn. They're not there to test compatibility of multiple versions of Tekton with different `apiVersion` versions; so IMO adding this as a parameter would not bring any benefits.